source/documentation/other-topics/custom-default-backend.md

```diff
@@ -113,7 +113,7 @@ spec:
   tls:
   - hosts:
     - helloworld.rubyapp.cloud-platform.service.justice.gov.uk
-    secretName: live0-to-live1-cert
+    secretName: secret-cert
```

Comment:
There's an earlier example file which still has `secretName: live0-to-live1-cert`

Please also make this change in that file, otherwise users might get confused and think that the secretName matters.